gravpm.c

```diff
@@ -299,6 +408,25 @@ void powerspectrum_compute(const int64_t k2, const int kpos[3], pfft_complex * c
 
 }
 
+/*Just read the power spectrum, without changing the input value.*/
+static void readout_power_spectrum(int64_t k2, int kpos[3], pfft_complex *value) {
```

Comment:
I have been reserving readout for configuration space resampling operations. What about call this measure_power_transfer?